woocommerce/woocommerce-ios
# Handled push notifications for multiple stores

This basic implementation allows the app to handle push notifications for multiple stores.
I edited the `StorePickerCoordinator`, and I implemented a static method that does under the hood the same things that the app does when we present to the user the store picker selector. The method has a callback, that is called when the store switch is completed. In the end, I use this method inside `MainTabBarController` which is the presenter of every push action.

Ref: p91TBi-2x0

## Testing
1) Follow this procedure p99K0U-1qz to configure your dev environment.
2) Launch the app on your device, subscribe the device for a specific store (remind it).
3) Close the app.
4) Open the code, comment the two methods `unregisterForRemoteNotifications()` and `registerDeviceToken()` in `PushNotificationsManager`.
5) Re-run the app, and change the store.
6) Fire a new push notification about an order for your initial store (the one subscribed for push notifications).
7) Check that you are able to see the order detail screen and that the app switched correctly the default store. Make sure that all the data inside the app are about the new store selected.
8) Repeat from step 6, but using a push notification for a new review.


Update release notes:
